sections/slideshow.liquid

```diff
@@ -73,7 +73,7 @@
           {%- endif -%}
         </div>
         <div class="slideshow__text-wrapper banner__content banner__content--{{ block.settings.box_align }} page-width{% if block.settings.show_text_box == false %} banner--desktop-transparent{% endif %}">
-          <div class="slideshow__text banner__box color-{{ block.settings.color_scheme }} slideshow__text--{{ block.settings.text_alignment }} slideshow__text-mobile--{{ block.settings.text_alignment_mobile }}">
+          <div class="slideshow__text banner__box content-containers content-containers--full-width-mobile color-{{ block.settings.color_scheme }} slideshow__text--{{ block.settings.text_alignment }} slideshow__text-mobile--{{ block.settings.text_alignment_mobile }}">
```

Comment:
Do you think that the bottom border should be rounded? Then it looks separate from the number/controls section.

<img width="709" alt="Screen Shot 2021-12-20 at 8 55 25 PM" src="https://user-images.githubusercontent.com/26778637/146857627-340c061c-f6c0-4f8a-912e-e2627b446c32.png">

Reply:
Yeah I think we might some UX exploration to see how this section should be affected by global settings. From the look of it I think it should be the controls that get a bottom border radius so the whole thing looks connected.

Reply:
Hey hey! Late to the party!

For `Slideshow` we have [2 potential solutions](https://screenshot.click/21-12-4wtnd-h7sgi.png).

- When the layout is set to `Grid`, it is the same outcome as an `Image with text` section but with a slider UI underneath. 
   - A: [We have the control UI border take the radius](https://screenshot.click/21-12-xkvfb-tvjze.png) at the bottom
   - B: We have the content container take the radius only at the bottom but we remove the control top border to [avoid this inconvenient gap](https://screenshot.click/21-12-kg0lw-28isz.png)
- In any case, when the layout is set to `Grid`, borders would be applied all around. Since the `Slideshow` asset isn't styled with the Media setting, we don't have to deal with adjacent borders like on `Image with text`.

Reply:
I think solution A looks the best 🤔  Maybe I will apply what you have there in this PR but we might need a follow up.

Should we also tweak the media a the top ? So it looks like this: 
![](https://screenshot.click/21-12-oywmm-ckd5q.png)

But should `content containers` be dictating what's happening for the whole slideshow ? 🤔

Reply:
Let's keep the application of the setting to the content container only for now 👍